components/metrics/metrics_log.cc

```diff
@@ -329,14 +329,16 @@ void MetricsLog::TruncateEvents() {
         internal::kUserActionEventLimit,
         uma_proto_.user_action_event_size() - internal::kUserActionEventLimit);
   }
-
+  // Omnibox proto removed for binary size reasons: b/290819695.
+#if !defined(STARBOARD)
```

Comment:
@andrewsavage1 do we do `STARBOARD` or `COBALT_CUSTOMIZATIONS` here ?

Reply:
`USE_COBALT_CUSTOMIZATIONS`

Reply:
When do I use which? E.g., in https://github.com/youtube/cobalt/pull/551/ I used STARBOARD everywhere and no one called it  out.

Reply:
Switched everything to `USE_COBALT_CUSTOMIZATIONS`

Reply:
Yeah that's exactly right, thanks.